charts/selenium-grid/values.yaml

```diff
@@ -398,6 +399,15 @@ chromeNode:
   #   persistentVolumeClaim:
   #     claimName: my-pv-claim
 
+  # Keda scaled object configuration
+  autoscalingEnabled: false
+  maxReplicaCount: 8
+  hpa:
+    url: http://selenium-hub.default:4444/graphql # Repalce your http graphql url here
+    browserName: chrome
+    # browserVersion: '91.0' # Optional. Only required when supporting multiple versions of browser in your Selenium Grid.
+    unsafeSsl : 'true' # Optional
```

Comment:
What does this mean?`unsafeSsl`?

Reply:
@diemol https://keda.sh/docs/2.8/scalers/selenium-grid-scaler/

"Skip certificate validation when connecting over HTTPS."